src/sslworld.cpp

```diff
@@ -504,7 +504,7 @@ void SSLWorld::recvActions()
                     bool wheels = false;
                     if (packet.commands().robot_commands(i).has_wheelsspeed())
                     {
-                        if (packet.commands().robot_commands(i).wheelsspeed()==true)
+                        if (packet.commands().robot_commands(i).wheelsspeed())
```

Comment:
I don't like this change, wheelspeed() is an auto generated function. This just checks if the field exists- not if it is false or true.

Reply:
Check, clang tidy was complaining I should remove ==true ;-)

I'll add it back in

Reply:
uhm I keep it in because it returns a bool

Reply:
Yes but then also refactor the rest of the file, this just feels pointless if you're going to refactor one == true when there is many in the whole file.

Reply:
anyways this doesn't matter too much, so I am approving, but If you're going to refactor you should do it on file-by-file  and not on a ' I saw these lines and c-lang complained' structure, in my opinion